regcomp.c

```diff
@@ -2281,8 +2281,8 @@ Perl_re_op_compile(pTHX_ SV ** const patternp, int pat_count,
         }
         if (0)
         for( int parno = 1; parno < RExC_total_parens ; parno++ )
-            PerlIO_printf(Perl_debug_log,"%d -> %d -> %d\n",
-                    parno, RExC_parno_to_logical[parno], RExC_parno_to_logical_next[parno]);
+            PerlIO_printf(Perl_debug_log,"%" IVdf " -> %" IVdf " -> %" IVdf "\n",
```

Comment:
Wouldn't it be better to use `I32df` instead of casting?

Reply:
It is debug code that I left in for myself to debug this feature if needed, notice the `if(0)`. I dont think it matters at all.

Reply:
Anyway, i changed the patch to just remove the entire if(0) function. Devel::Peek has this now.